src/main/resources/META-INF/spring.factories

```diff
@@ -1,2 +1,2 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-  uk.gov.hmcts.reform.ccd.client.healthcheck.CoreCaseDataHealthIndicator
+  uk.gov.hmcts.reform.ccd.client.CoreCaseDataClientConfiguration
```

Comment:
I would name class `CoreCaseDataClientAutoConfiguration` as this seems to be spring convention.

See fo reference:

https://github.com/spring-projects/spring-boot/blob/v1.5.9.RELEASE/spring-boot-autoconfigure/src/main/resources/META-INF/spring.factories